test/parallel/test-process-kill-pid.js

```diff
@@ -57,16 +57,19 @@ assert.throws(function() { process.kill(1 / 0); },
 assert.throws(function() { process.kill(-1 / 0); },
               invalidPidArgument);
 
-// Test that kill throws an error for invalid signal
-const unknownSignal = common.expectsError({
+// Test that kill throws an error for unknown signal names
+common.expectsError(process.kill(1, 'test'), {
```

Comment:
`common.expectsError(
  () => process.kill(1, 'test'),
  { ... }
`

That is, The first argument needs to be a function...